pkg/environment/environment_test.go

```diff
@@ -15,13 +15,57 @@
 package environment
 
 import (
+	"os"
 	"path/filepath"
+	"strings"
 	"testing"
 )
 
+func TestMustGetKrewPaths_resolvesToHomeDir(t *testing.T) {
+	home := os.Getenv("USERPROFILE")
+	if home == "" {
+		home = os.Getenv("HOME")
+	}
+	if home == "" {
+		t.Fatal("cannot determine HOME or USERPROFILE")
+	}
+	expectedBase := filepath.Join(home, ".krew")
+
+	p := MustGetKrewPaths()
+	if got := p.BasePath(); got != expectedBase {
+		t.Fatalf("MustGetKrewPaths()=%s; expected=%s", got, expectedBase)
+	}
+}
+
+func TestPaths(t *testing.T) {
+	base := filepath.FromSlash("/foo")
+	p := newPaths(base)
+	if got := p.BasePath(); got != base {
```

Comment:
When comparign paths we should compare the clean paths: `filepath.Clean(got) != filepath.Clean(want)`

Reply:
Hmm why? If a=b, then f(a)=f(b), too. And right now tests are passing for a=b.

Reply:
We can have the case that `"foo/bar" != "foo/bar/"` but it should be, but
`filepath.Clean("foo/bar") == filepath.Clean("foo/bar/")`.

Reply:
I prefer this method not return a trailing slash. Adding this makes the test more forgiving and there’s no reason to be more forgiving now.